### PR TITLE
HDDS-2065. Implement OMNodeDetails#toString

### PR DESCRIPTION
**Pull request by smengcl**

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMNodeDetails.java [hunk 1]
```diff
@@ -54,6 +54,19 @@ private OMNodeDetails(String serviceId, String nodeId,
...
   }
 
+  @Override
+  public String toString() {
+    return "OMNodeDetails["
+        + "omServiceId=" + omServiceId +
+        ", omNodeId=" + omNodeId +
+        ", rpcAddress=" + rpcAddress +
+        ", rpcPort=" + rpcPort +
+        ", ratisPort=" + ratisPort +
```
Comment (bharatviswa504) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMNodeDetails.java]: Minor suggestion: Use StringBuilder instead of using "+".

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 40 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 590 | trunk passed |
| +1 | compile | 380 | trunk passed |
| +1 | checkstyle | 83 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 879 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 178 | trunk passed |
| 0 | spotbugs | 421 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 615 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 536 | the patch passed |
| +1 | compile | 386 | the patch passed |
| +1 | javac | 386 | the patch passed |
| +1 | checkstyle | 89 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 675 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 176 | the patch passed |
| +1 | findbugs | 625 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 266 | hadoop-hdds in the patch passed. |
| -1 | unit | 1860 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 56 | The patch does not generate ASF License warnings. |
| | | 7611 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures |
|   | hadoop.ozone.client.rpc.TestFailureHandlingByClient |
|   | hadoop.ozone.container.common.statemachine.commandhandler.TestBlockDeletion |
|   | hadoop.ozone.om.TestSecureOzoneManager |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1387/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1387 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux f9e4f2b5182b 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 915cbc9 |
| Default Java | 1.8.0_222 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1387/1/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1387/1/testReport/ |
| Max. process+thread count | 5404 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/ozone-manager U: hadoop-ozone/ozone-manager |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1387/1/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by bharatviswa504** [comment]
Thank You @elek for sharing info.

**Pull request closed without merging**